app/views/layouts/application.html.erb

```diff
@@ -4,6 +4,7 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <%= render partial: 'meta_tags' %>
```

Comment:
__[rails_best_practices]__
* simplify render in views [:ref](http://rails-bestpractices.com/posts/2010/12/04/simplify-render-in-views/)